utils/config.env

```diff
@@ -1,6 +1,28 @@
 #####################################################
-# Values in this file will be used to replace variables in script files
-# and Docker Compose files throughout the repo
+# The following values are provided by Confluent's
+# release automation process.  The values in these
+# names will be adopted in scripts and expressions
+# over time to construct other
+# values required by this repository
+#####################################################
+CONFLUENT=5.4.1-SNAPSHOT
+CONFLUENT_DOCKER_TAG=5.4.x-latest
+CONFLUENT_SHORT=5.4
+CONFLUENT_PREVIOUS=5.3.0
```

Comment:
Not a big deal but IIUC this should be `5.4.0`

Reply:
It will actually be an empty string if its a version that ends in a `.0`

Reply:
So then what @ybyzek says is correct... if this is 5.4.1 then PREVIOUS is 5.4.0?

Reply:
Yes